modules/search/class.jetpack-search.php

```diff
@@ -380,7 +414,7 @@ public function get_es_wp_query_terms_for_query( WP_Query $query ) {
 
 		foreach ( $the_tax_query->queries as $tax_query ) {
 			// Right now we only support slugs...see note above
-			if ( 'slug' !== $tax_query['field'] ) {
+			if ( ! is_array( $tax_query ) || 'slug' !== $tax_query['field'] ) {
```

Comment:
@nickdaugherty can you take a look at this?

Reply:
Looks good to me. @gravityrail's commit message about tax query relations makes me realize that we may not be handling AND/OR correctly, but that's a separate issue.